src/main/java/org/schemaspy/connection/Filed.java

```diff
@@ -0,0 +1,35 @@
+package org.schemaspy.connection;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Properties from this file (in key=value pair format) are passed to the
+ * database connection.<br>
+ * user (from -u) and password (from -p) will be passed in the
+ * connection properties if specified.
+ */
+public class Filed {
```

Comment:
Filed is a term for archiving, this is more of a PropertiesFromFile.

Which more or less probably can be abstracted to some kind of Input.
Since the Inputstream from a file could look the same as a string made into an inputstream.

Reply:
`PropertiesFromFile` is a bit long, but otherwise yes. What about `FromFile`?
Generalization is a later step that might not be worthwhile.

Reply:
It's a bit long but, the Object has multiple responsibilities so I think it's fitting.

Wanna make it shorter, remove some of it's responsibilities.